class A{
 void foo() {
  throw new OurNotException.InnerException()<caret>
 }
}